lib/rubyfocus/fetchers/local_fetcher.rb

```diff
@@ -45,7 +45,7 @@ def patches
 	# Fetches the contents of a given patch file
 	def patch(file)
 		filename = File.join(self.location, file)
-		if File.exists?(filename)
+		if File.exist?(filename)
```

Comment:
`File.exists?` was [removed](https://bugs.ruby-lang.org/issues/17391) in Ruby 3.2.